features/steps/steps.py

```diff
@@ -79,6 +79,19 @@ class instance_structure_error:
     def __str__(self):
         return f"The instance {fmt(self.related)} is assigned to {fmt(self.relating)}"
 
+@dataclass
+class instance_contained_error:
```

Comment:
We should probably merge this with @Ghesselink's `instance_structure_error` https://github.com/buildingSMART/ifc-gherkin-rules/pull/6/files#diff-cf027576a6812e3403bd3d9579d83abfaa74a41607a4f3c986398ef8492a5d3dR74